src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingService.kt

```diff
@@ -244,6 +248,7 @@ class SentencingService(
               },
             )
           }
+          storedProcedureRepository.imprisonmentStatusUpdate(booking.bookingId)
```

Comment:
idempotent SP - not too worried about detecting if court event charge has actually changed as not a big overhead